nevergrad/functions/unit_commitment/core.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
+import nevergrad as ng
+from ..base import ExperimentFunction
+
+class UnitCommitmentProblem1(ExperimentFunction):
+    """Model that uses conventional implementation for emi-continuous variables
+    The model is adopted from Pyomo model 1: Conventional implementation for emi-continuous variables
+    (https://jckantor.github.io/ND-Pyomo-Cookbook/04.06-Unit-Commitment.html)
+    The constraints are added to the objective with a heavy penalty for violation.
+
+    Parameters
+    ----------
+    T_points: int
+        number of time points
+    N_generators: int
+        number of generators
+    penalty_weight: float
+        weight to penalize for violation of constraints
+    """
+    def __init__(self, T_points: int = 13,
+                 N_generators: int = 3,
```

Comment:
for pep8 compliance please rename without capital letters.
I would recommend `num_timepoints` and `num_generators` for clarity, or even just `timepoints` and `generators` since fro; the typing it's clear that it's a number